_pages/business-units/18f/how-18f-works/project-lifecycle.md

```diff
@@ -0,0 +1,88 @@
+---
+title: High-level project lifecycle
```

Comment:
```suggestion
title: 18F project lifecycle
```

Reply:
I assumed being under the 18F section, in the "How We Work" subsection would obviate the need for "18F" in the page title, but if that's a convention, I'm happy to make this change. I was worried that just "project lifecycle" might imply a greater level of detail than the document intends to provide.

Reply:
It's under the 18F section in terms of the folder and the navigation (on the front page), but that isn't clear when you're on the page itself. Until we have breadcrumbs or something, I suggest we have the title make it clear when pages are scoped to certain teams. "18F high-level project lifecycle"?

Reply:
> Until we have breadcrumbs or something

I think I'd agree for now, but I do wonder whether having a more sensible file path would alleviate your concern...?

Reply:
Ah, I see where you're coming from. Would changing the filename to `18f-project-lifecycle.md` be suitable, then? I just checked and other 18F-specific pages don't have "18F" in the title, so I'm not sure I want to start that trend if there's a longer-term goal of breadcrumbing or similar.

Reply:
> I just checked and other 18F-specific pages don't have "18F" in the title

It's a problem 😛 

Another option is making the URL `/18f/project-lifecycle/`.

@dylanirlbeck I'm going to defer to you to make the call on what we go with for now. (It's reversible: titles can be easily changed, redirects can be made.)

Reply:
@mgwalker Yeah I think the best option for now is to set the title and filename to '18F project lifecycle' and `18f-project-lifecycle.md`, respectively. 

I see two options for changing this down the road: (1) changing the folder structure and Jekyll rule so that the full path is visible (and hence 18F would show up in the URL) and (2) adding some sort of indicator to pages that only apply to certain teams (like 18F in this case).

Reply:
Renamed to `18f-project-lifecycle.md` and made the title `High-level 18F project lifecycle`